traktapi.py

```diff
@@ -332,6 +332,10 @@ def getShowSummary(self, showId):
         with Trakt.configuration.http(retry=True):
             return Trakt['shows'].get(showId)
 
+    def getShowWithAllEpisodesList(self, showId):
+        with Trakt.configuration.http(retry=True):
```

Comment:
You might want to add a timeout here, as this is a read call `(retry=True, timeout=90):`